README.md

```diff
@@ -72,6 +72,8 @@ See <https://docs.celestia.org/category/celestia-app> for more information
 
 1. Install [golangci-lint](https://golangci-lint.run/usage/install/)
 1. Install [markdownlint](https://github.com/DavidAnson/markdownlint)
+1. Install [hadolint](https://github.com/hadolint/hadolint)
+1. Install [yamllint](https://yamllint.readthedocs.io/en/stable/quickstart.html)
```

Comment:
nit: shouldn't the numbers be incrementing (or why are they there)

Reply:
We could also do something like https://github.com/cometbft/cometbft/blob/main/tools/tools.go which, IIUC, will install these tools so developers don't need to manually install them

Reply:
Markdown will render an ordered list with all leading `1.` with the correct number. This make it easy to re-arrange the lines in the list without manually updating each number.

Ref: https://www.markdownguide.org/basic-syntax/#ordered-lists

The idea from cometbft sounds good to me. Unfortunately in the list above, only golangci-lint is written in Go.

Reply:
> Markdown will render an ordered list with all leading 1. with the correct number. This make it easy to re-arrange the lines in the list without manually updating each number.

TIL